src/runtime/uv/timer.cpp

```diff
@@ -235,19 +233,27 @@ extern "C" LEAN_EXPORT lean_obj_res lean_uv_timer_stop(b_obj_arg obj, obj_arg /*
 void lean_uv_timer_finalizer(void* ptr);
 
 extern "C" LEAN_EXPORT lean_obj_res lean_uv_timer_mk(uint64_t timeout, uint8_t repeating, obj_arg /* w */) {
-    return io_result_mk_error("lean_uv_timer_mk is not supported");
+    lean_always_assert(
+        false && ("Please build a version of Lean4 with libuv to invoke this.")
+    );
 }
 
 extern "C" LEAN_EXPORT lean_obj_res lean_uv_timer_next(b_obj_arg timer, obj_arg /* w */ ) {
-    return io_result_mk_error("lean_uv_timer_next is not supported");
+        _always_assert(
```

Comment:
You are using `lean_always_assert` in two of these functions and `_always_assert` in the other two.